### PR TITLE
fixed problems

### PR DESCRIPTION
**Pull request by hsnbozkurt**

### CODE REVIEW

**Comment by SadMap** [comment]
@DeltaCoderr  can u look ?

File: package.json [hunk 1]
```diff
@@ -4,13 +4,8 @@
...
-     "discord.js": "^12.3.1",
-    "fs": "0.0.1-security",
-     "path": "^0.12.7",
-     "quick.db": "^7.1.2"
-   },
-  "engines": {
-    "node": "12.x"
+    "@hapi/code": "^8.0.3",
+    "discord.js": "^12.5.1",
+    "quick.db": "^7.1.3"
```
Comment (DeltaCoderr) [inline package.json:9]: We require `path` and `fs` packages here as well.

**Review by sbdh11 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (SadMap) [reply package.json:9]: i tested and i think this packages are bundeled with node becuse i deleted node_modules and installed required packsges woth --save option

Comment (DeltaCoderr) [reply package.json:9]: Makes sense, Alright then.

**Pull request merged**